Assets/Scripts/GameSelection.cs

```diff
@@ -1,13 +1,17 @@
-﻿using UnityEngine;
-using System.Collections;
+﻿using System;
```

Comment:
Haven't tested this but I am avoiding using System as there are many conflicts between System and UnityEngine.  Where I need system I just specify explicitly.